client/main/tableview.js

```diff
@@ -761,16 +767,82 @@ const TableView = SilkyView.extend({
                     case 'ordinal':
                         if (Number.isInteger(number))
                             value = number;
-                        else if ( ! this.currentColumn.autoMeasure)
-                            throw {
-                                title: 'Integer value required',
-                                message: 'Nominal and Ordinal variables only accept integer values',
-                                type: 'error'
-                            };
+                        else if ( ! this.currentColumn.autoMeasure) {
+                            let found = false;
+
+                            if (typeof value === 'string') {
+                                for (let levelInfo of this.currentColumn.levels) {
+                                    if (value === levelInfo.label) {
+                                        value = levelInfo.value;
+                                        found = true;
+                                        break;
+                                    }
+                                }
+                            }
+
+                            if ( ! found) {
+
+                                return new Promise((resolve, reject) => {
+                                    keyboardJS.pause();
+                                    this._editing = false;
+                                    setTimeout(() => {
+                                        dialogs.prompt('Enter an integer value for the label \'' + value + '\'', '', (result) => {
```

Comment:
sorry for stuffing you around, but we're now thinking we should just assign an integer value automatically. i.e. pick the next one available.